main/src/ca/uwaterloo/flix/tools/Packager.scala

```diff
@@ -481,11 +483,22 @@ object Packager {
     addToZip(zip, name, Files.readAllBytes(p))
   }
 
+  /**
+    * To support DOS time, Java 8+ treats dates before the 1980 January in special way.
+    * Here we use 1980 February to avoid the complexity introduced by this hack.
+    *
+    * @see <a href="https://bugs.openjdk.java.net/browse/JDK-4759491">JDK-4759491 that introduced the hack around 1980 January from Java 8+</a>
+    * @see <a href="https://bugs.openjdk.java.net/browse/JDK-6303183">JDK-6303183 that explains why the second should be even to create ZIP files in platform-independent way</a>
+    * @see <a href="https://github.com/gradle/gradle/blob/445deb9aa988e506120b7918bf91acb421e429ba/subprojects/core/src/main/java/org/gradle/api/internal/file/archive/ZipCopyAction.java#L42-L57">A similar case from Gradle</a>
+    */
+  private val ENOUGH_OLD_CONSTANT_TIME: Long = new GregorianCalendar(1980, Calendar.FEBRUARY, 1, 0, 0, 0).getTimeInMillis
+
   /**
     * Adds an entry to the given zip file.
     */
   private def addToZip(zip: ZipOutputStream, name: String, d: Array[Byte]): Unit = {
     val entry = new ZipEntry(name)
+    entry.setTime(ENOUGH_OLD_CONSTANT_TIME)
```

Comment:
At a high level, is there any danger to overriding the time like this?

Reply:
I think so.  
At least for me, it is difficult to imagine that someone has an operation depending on the time of zip enties.

Reply:
Would it be more sensible to use a newer date? Let's say 1 Jan 2000 or 2020? 

I just wonder if some systems would find it strange with such an old date.

Reply:
Or some symbolic date as an Easter egg, like the date of the first commit to Flix.

Though maybe it's better for the date to be clearly generic.

Reply:
I think it is OK to use another date. How about `2014-06-27` the date of the first commit (69852b75c7875d25ae9e8fef2e9bdd8cdf19ce7d) to Flix?

Reply:
Sure. OK